addons/dark-www/addon.json

```
@@ -14,6 +14,12 @@
       "note": "Dark WWW",
       "id": "darkwww",
       "link": "https://github.com/gducrash"
+    },
+    {
+      "name": "Valer100",
+      "note": "Some fixes",
+      "id": "fixes",
+      "link": "https://github.com/Valer100"
```

I wouldn't call these changes "fixes" because the previous behavior wasn't wrong.
```suggestion
    {
      "name": "Valer100",
      "link": "https://github.com/Valer100"
```